docs/guides/agent/agent_component_reference/message.md

```
@@ -30,4 +30,12 @@ Click **+ Add message** to add message options. When multiple messages are suppl
 Save the conversation to specified memories. Expand the dropdown list to either select all available memories or specified memories:
 
 
-![](https://raw.githubusercontent.com/infiniflow/ragflow-docs/main/images/save_to_memory.png)
\ No newline at end of file
+![](https://raw.githubusercontent.com/infiniflow/ragflow-docs/main/images/save_to_memory.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)